### PR TITLE
completed question 3

### PR DESCRIPTION
**Pull request by SijoGeorge7012**
- [x] Created a method Sort with int[] returntype and int[] parameter.

- [x] Created 10 Test method for method Sort.

image

### CODE REVIEW

File: C#/cs01/03.cs [hunk 1]
```diff
@@ -4,8 +4,23 @@ namespace Brainchild.Curriculum.Solutions
...
+			    {
+				   if (arr[j+1] < arr[j])
+				    {
+				    	temp = arr[j];
+					    arr[j] = arr[j + 1];
+					    arr[j + 1] = temp;
+				    }
+			    }
+		    }
+            return arr;
```
Comment (aneeshs005) [inline C#/cs01/03.cs:22]: Indentation problem

File: C#/cs01/03.cs [hunk 2]
```diff
@@ -4,8 +4,23 @@ namespace Brainchild.Curriculum.Solutions
...
         }
+        
     }
-}
\ No newline at end of file
+}
```
Comment (aneeshs005) [inline C#/cs01/03.cs:25]: Add new line at the end

File: C#/cs01/03.cs [hunk 3]
```diff
@@ -4,8 +4,23 @@ namespace Brainchild.Curriculum.Solutions
...
+        {
+		    int temp;
+		    for (int i = 0; i < arr.Length; i++)
+		    {
+			   for (int j =0; j < arr.Length-1; j++)
+			    {
+				   if (arr[j+1] < arr[j])
+				    {
+				    	temp = arr[j];
+					    arr[j] = arr[j + 1];
```
Comment (aneeshs005) [inline C#/cs01/03.cs]: Indentation problem

**Review by aneeshs005 — CHANGES_REQUESTED** [review]
Make required changes and push again

File: C#/cs01/03.cs (hunk 2 above)
Comment (SijoGeorge7012) [reply C#/cs01/03.cs:25]: - [x] i already add it

image

**Pull request merged**